tests/TestData/success.json

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
Good call to make full mock files for these. However as we're making a global home for test data, I think this needs more specific naming so it's clear what it's for. The type of hook should be in a folder name or the file name. Same with the others.